docs/src/pages/components/data-grid/group-pivot/group-pivot.md

```diff
@@ -61,6 +61,22 @@ Use the `groupingColDef` prop to customize the rendering of the grouping column.
 
 {{"demo": "pages/components/data-grid/group-pivot/CustomGroupingColumnTreeData.js", "bg": "inline", "defaultCodeOpen": false}}
 
+#### Accessing the grouping column field
+
+If you want to access the grouping column field, for instance to pin it, you can use the `GRID_TREE_DATA_GROUPING_FIELD` constant.
```

Comment:
Explicitly writing "column pinning" makes easier to find with <kbd>CTRL</kbd> + <kbd>f</kbd>. 

```suggestion
If you want to access the grouping column field, for instance, to use it with column pinning, the `GRID_TREE_DATA_GROUPING_FIELD` constant is available.
```